requirements.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - clang=16
+  - clang-format
```

Comment:
clang-format also needs to be 16

Reply:
Indeed, overlooked it. Btw, it seems to resolve to v16 because clang is v16.